ngrinder-frontend/src/js/components/perftest/mixin/FormatMixin.vue

```diff
@@ -28,7 +28,10 @@
         }
 
         formatPercentage(format, value) {
-            value = value || 0;
+            if (!value) {
+                return 'N/A';
```

Comment:
What about give space after A like ``` 'N/A '```

Reply:
I tried to add a space after it but, this value is wrapped by `<span/>` tag. So additional white space doesn't affect to html.
![image](https://user-images.githubusercontent.com/10591327/63573867-6f68a600-c5c1-11e9-93a1-d964f0235267.png)